projects/plugins/jetpack/tools/check-block-assets.php

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Script to check blocks' view.asset.php for unexpected dependencies.
+ *
+ * @package automattic/jetpack
+ */
+
+// phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped -- This is not WordPress code.
+
+/**
+ * Non-WP dependencies to check for.
+ *
+ * @var string[]
+ */
+$bad_deps = array(
```

Comment:
This is all the non-WP scripts externalized by `@wordpress/dependency-extraction-webpack-plugin`, since all seem large.